core-bundle/assets/controllers/autofocus-controller.js

```diff
@@ -0,0 +1,29 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['input']
+
+    connect () {
+        this.inputTargets.every(input => {
```

Comment:
Instead of iterating over every `inputTarget` in connect, you can also just use

```js
inputTargetConnected(input) {
    …
}
```

This would preferable as this would be executed immediately when the `input` is added to the DOM.

Reply:
The script is supposed to stop after the first matching input field. How would that work with `inputTargetConnected`?

Reply:
You could track that with a static variable - but it's probably not worth it.